src/main/java/org/myrobotlab/kinematics/DHRobotArm.java

```diff
@@ -235,7 +235,10 @@ public boolean moveToGoal(Point goal) {
     int numSteps = 0;
     double iterStep = 0.05;
     // we're in millimeters..
-    double errorThreshold = 2.0;
+    double errorThreshold = 20.0;
+    
+    maxIterations = 1000;

```

Comment:
if you want a different default value for this it should be specified above, not overridden here.

Reply:
reverting